kitsune/wiki/models.py

```diff
@@ -990,7 +990,7 @@ def previous(self):
             return None
 
 
-class DraftRevision(ModelBase, AbstractRevision):
+class DraftRevision(ModelBase, AbstractRevision):  # type: ignore
```

Comment:
This is required due to another strange `mypy` behavior. It complains that the `ModelBase` and `AbstractRevision` classes have incompatible `Meta` class attributes, when in fact they're both exactly the same. It's probably due to Django's special handling of `Meta` class attributes (it sets `Meta.abstract` to `False` for all sub-classes).